### DISCUSSION TITLE
docs/v1.11.x/ops/kubectl-ko/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#3057] [Announcements]
# docs/v1.11.x/ops/kubectl-ko/

https://kubeovn.github.io/docs/v1.11.x/ops/kubectl-ko/

**Comment by xinsir88** [comment]
大佬好，因为我们公司内网没有互联网，所有想把咱们得帮助文档搬到我们内网，我该如何操作呢？
有没有类似k8s的官方文档的website那种的服务， 一下直接搞过去呢？求指导。

**Reply by oilbeater** [reply]
可以参考 https://github.com/kubeovn/docs 里面 .github 下的 workflow

**Reply by xinsir88** [reply]
老师，看了下哪个问题，没有太好的思路，能不能给详细解释下呢？

**Reply by xinsir88** [reply]
搞定啦，感谢老师。

**Comment by xinsir88** [comment]
顶一下子。
